src/pages/PersonalTraining/PersonalTraining.jsx

```diff
@@ -1,7 +1,178 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import ContentTab from '../../components/ContentTab/ContentTab';
+import ProfileListBox from './components/ProfileListBox/ProfileListBox';
+import PersonalTrainingBox from './components/PersonalTrainingBox/PersonalTrainingBox';
+import DaysUntilStart from './components/DaysUntilStart/DaysUntilStart';
+import SelectStartDate from './components/SelectStartDate/SelectStartDate';
+import './PersonalTraining.scss';
+import 'react-calendar/dist/Calendar.css';
 
 const PersonalTraining = () => {
-  return <div>PersonalTraining</div>;
+  const [membershipData, setMembershipData] = useState({});
+  const startDate = new Date(`${membershipData[0]?.start_date}`);
+
+  const startYearMonthDate = new Date(
+    startDate.getFullYear(),
+    startDate.getMonth(),
+    startDate.getDate(),
+  );
+
+  const nowDate = new Date();
+
+  const nowYearMonthDate = new Date(
+    nowDate.getFullYear(),
+    nowDate.getMonth(),
+    nowDate.getDate(),
+  );
+  const [ptStartDate, setPtStartDate] = useState('');
+  const week = makeWeekArr(nowYearMonthDate);
+  const [selectedDate, setSelectedDate] = useState(`${nowYearMonthDate}`);
+  const [selectedTab, setSelectedTab] = useState(1);
+  const [trainerData, setTrainerData] = useState();
+  const [dateState, setDateState] = useState({
+    date: nowYearMonthDate,
+    week: week,
+  });
+  const [checkedMyExercise, setCheckedSetMyExercise] = useState([]);
+  const [checkedMyDiet, setCheckedMyDiet] = useState([]);
+  const [selectedButton, setSelectedButton] = useState();
+
+  useEffect(() => {
+    if (!ptStartDate) {
+      setSelectedButton(true);
+    } else {
+      setSelectedButton(false);
+    }
+  }, [ptStartDate]);
```

Comment:
state가 아닌 변수로 만들어서 사용하면 될 것 같습니다.
```suggestion
  const isSelected = !ptStartDate;
```